datasets/imdb/README.md

```diff
@@ -19,6 +19,55 @@ task_categories:
 task_ids:
 - sentiment-classification
 paperswithcode_id: imdb-movie-reviews
+train-eval-index:
+- config: plain_text
+  task: text-classification
+  task_id: binary_classification
```

Comment:
for reference, the task_id is the autotrain one, they're defined here (private) : https://github.com/huggingface/autonlp-backend/tree/main/zeus/zeus/models